playbooks/dnsrecord/ensure-dnsrecord-is-absent.yml

```diff
@@ -1,12 +1,12 @@
 ---
-- name: Test DNS Record is absent.
+- name: Playbook to manage DNS records.
   hosts: ipaserver
   become: true
   gather_facts: false
 
   tasks:
-  # Ensure that dns record is absent
-  - ipadnsrecord:
+  - name: Ensure that dns record is absent
```

Comment:
I think something like `Ensure that dns localhost AAAA record is absent` would be better.

Reply:
Fixed.